### PR TITLE
update(Language): Add spanish

### LINKED ISSUE #119: Make languages dropdown more understandable
**Issue:** In Settings / General when someone opens to dropdown menu for languages its a little difficult to understand for an average user. 

**Screenshot:**

Screenshot 2023-01-11 at 1 07 48 PM

### PR DESCRIPTION
**Pull request by lgmarchi**
### 2. Add more friendly language names 
image

### Which issue(s) this PR fixes 🔨

- Resolve #119

### CODE REVIEW

**Comment by sdwoodbury** [comment]
this looks good; PR #114 should probably be merged first

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Good for me but gonna wait for @luisecm 👍

File: shared/locales/es-MX/main.ftl [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    .call = Llamada
+    .video-call = Video-llamada
+    .chat = Chat
+    .chats = Chats
+    .send = Enviar
+    .remove = Eliminar
+    .more = Más
+    .welcome = Bienvenido
+    .clear-unreads = Limpiar no leídos
+    .hide-chat = Ocultar chat
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .hide-chat = Ocultar Chat
```

File: shared/locales/es-MX/main.ftl [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    .add = Añadir
+    .call = Llamada
+    .video-call = Video-llamada
+    .chat = Chat
+    .chats = Chats
+    .send = Enviar
+    .remove = Eliminar
+    .more = Más
+    .welcome = Bienvenido
+    .clear-unreads = Limpiar no leídos
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .clear-unreads = Limpiar No Leídos
```

File: shared/locales/es-MX/main.ftl [hunk 3]
```diff
@@ -0,0 +1,152 @@
+uplink = Uplink
+    .home = Inicio
+    .add = Añadir
+    .call = Llamada
+    .video-call = Video-llamada
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .video-call = Videollamada
```

File: shared/locales/es-MX/main.ftl [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    .send = Enviar
+    .remove = Eliminar
+    .more = Más
+    .welcome = Bienvenido
+    .clear-unreads = Limpiar no leídos
+    .hide-chat = Ocultar chat
+    .search-placeholder = Buscar...
+    .end = Finalizar
+    .pre-release = Pre-lanzamiento
+    .username = Nombre de usuario
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .username = Usuario
```

File: shared/locales/es-MX/main.ftl [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    .status = Mensaje de estado
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .status = Estado
```

File: shared/locales/es-MX/main.ftl [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+    .maximum-of = Máximo de
+    .only-alpha-chars = Solo se aceptan caracteres alfanuméricos.
+    .spaces-not-allowed = Los espacios no están permitidos.
+
+messages = Messages
+    .new = Nuevo Mensaje
+    .replying = Respondiendo a:
+    .react = Reaccionar
+    .reply = Responder
+    .you = Tu
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .you = Tú
```

File: shared/locales/es-MX/main.ftl [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+    .search-placeholder = Buscar...
+    .end = Finalizar
+    .pre-release = Pre-lanzamiento
+    .username = Nombre de usuario
+    .status = Mensaje de estado
+    .characters = Caracteres
+    .exceeded = Excedido
+
+warning-messages = Warning Messages
+    .please-enter-at-least = Por favor ingrese al menos
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .please-enter-at-least = Por favor, ingrese al menos
```

File: shared/locales/es-MX/main.ftl [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+    .new = Nuevo Mensaje
+    .replying = Respondiendo a:
+    .react = Reaccionar
+    .reply = Responder
+    .you = Tu
+    .say-something-placeholder = Di algo...
+
+favorites = Favorites
+    .favorites = Favoritos
+    .add = Añadir a favoritos
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .add = Añadir a Favoritos
```

File: shared/locales/es-MX/main.ftl [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+    .remove = Eliminar de favoritos
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .remove = Eliminar de Favoritos
```

File: shared/locales/es-MX/main.ftl [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+friends = Friends
+    .friends = Amigos
+    .cta-text = Aún no tienes amigos. ¡Agrega algunos!
+    .add = Agregar una persona
+    .placeholder = Nombre de usuario#0000...
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .placeholder = Usuario#0000...
```

File: shared/locales/es-MX/main.ftl [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+    .block = Bloquear
+    .all = Todos los amigos
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .all = Todos Tus Amigos
```

File: shared/locales/es-MX/main.ftl [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+    .remove = Eliminar amistad
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .remove = Eliminar Amistad
```

File: shared/locales/es-MX/main.ftl [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+    .incoming_requests = Solicitudes entrantes
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .incoming_requests = Solicitudes Recibidas
```

File: shared/locales/es-MX/main.ftl [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+    .outgoing_requests = Solicitudes salientes
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .outgoing_requests = Solicitudes Enviadas
```

File: shared/locales/es-MX/main.ftl [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+    .cancel = Cancelar solicitud
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:51]:
```suggestion
    .cancel = Cancelar Solicitud
```

File: shared/locales/es-MX/main.ftl [hunk 16]
```diff
@@ -0,0 +1,152 @@
...
+    .requested = Solicitada
+    .sent = Enviada
+    .requests = Solicitudes
+    .pending = Pendiente
+    .blocked = Bloqueado
+    .accept = Aceptar
+    .deny = Rechazar solicitud
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:58]:
```suggestion
    .deny = Rechazar Solicitud
```

File: shared/locales/es-MX/main.ftl [hunk 17]
```diff
@@ -0,0 +1,152 @@
...
+
+files = Files
+    .files = Archivos
+    .new-folder = Nueva carpeta
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:62]:
```suggestion
    .new-folder = Nueva Carpeta
```

File: shared/locales/es-MX/main.ftl [hunk 18]
```diff
@@ -0,0 +1,152 @@
...
+    .upload = Cargar
+    .free-space = Espacio libre:
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:64]:
```suggestion
    .free-space = Espacio Libre:
```

File: shared/locales/es-MX/main.ftl [hunk 19]
```diff
@@ -0,0 +1,152 @@
...
+    .total-space = Espacio total:
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:65]:
```suggestion
    .total-space = Espacio Total:
```

File: shared/locales/es-MX/main.ftl [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+settings = Settings
+    .settings = Configuración
+    .profile = Perfil
+    .general = General
+    .files = Archivos
+    .privacy = Privacidad
+    .audio = Audio
+    .extensions = Extensiones
+    .developer = Desarrollador
+    .search-placeholder = Buscar configuraciones...
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:76]:
```suggestion
    .search-placeholder = Buscar Configuraciones...
```

File: shared/locales/es-MX/main.ftl [hunk 21]
```diff
@@ -0,0 +1,152 @@
...
+
+settings-profile = Profile Settings
+    .change-banner = Cambiar banner
+    .change-avatar = Cambiar avatar
+    .edit-button = Editar
+    .save-button = Guardar
+
+settings-general = General Settings
+    .overlay = Superposición Uplink
+    .overlay-description = Habilitar la superposición de pantalla Uplink. Esto mostrará la información de llamadas activas y también te permitirá agregar widgets personalizados a tu pantalla.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:86]:
```suggestion
    .overlay-description = Habilitar la superposición de pantalla para Uplink. Esto mostrará la información de llamadas activas, así como agregar widgets personalizados a tu pantalla.
```

File: shared/locales/es-MX/main.ftl [hunk 22]
```diff
@@ -0,0 +1,152 @@
...
+    .splash-screen = Pantalla de inicio
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:87]:
```suggestion
    .splash-screen = Pantalla de Inicio
```

File: shared/locales/es-MX/main.ftl [hunk 23]
```diff
@@ -0,0 +1,152 @@
...
+    .splash-screen-description = Deshabilitar la pantalla de inicio puede acelerar los tiempos de carga.
+    .app-language = Idioma de la aplicación
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:89]:
```suggestion
    .app-language = Idioma de la Aplicación
```

File: shared/locales/es-MX/main.ftl [hunk 24]
```diff
@@ -0,0 +1,152 @@
...
+    .change-language = Cambiar idioma
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .change-language = Cambiar Idioma
```

File: shared/locales/es-MX/main.ftl [hunk 25]
```diff
@@ -0,0 +1,152 @@
...
+    .theme = Tema
+    .theme-description = Cambia el tema de la aplicación
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:92]:
```suggestion
    .theme-description = Cambia el tema de la aplicación.
```

File: shared/locales/es-MX/main.ftl [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+    .theme-reset = Restaurar tema
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:93]:
```suggestion
    .theme-reset = Restaurar Tema
```

File: shared/locales/es-MX/main.ftl [hunk 27]
```diff
@@ -0,0 +1,152 @@
...
+    .theme-reset-description = Restaurar el tema predeterminado
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:94]:
```suggestion
    .theme-reset-description = Restaurar el tema a predeterminado
```

File: shared/locales/es-MX/main.ftl [hunk 28]
```diff
@@ -0,0 +1,152 @@
...
+    .theme-reset-cta = Limpiar tema
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:95]:
```suggestion
    .theme-reset-cta = Limpiar Tema
```

File: shared/locales/es-MX/main.ftl [hunk 29]
```diff
@@ -0,0 +1,152 @@
...
+
+settings-extensions = Extension Settings
+    .placeholder = Marcador de posición
+    .open-extensions-folder = Abrir carpeta de extensiones
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:99]:
```suggestion
    .open-extensions-folder = Abrir Carpeta de Extensiones
```

File: shared/locales/es-MX/main.ftl [hunk 30]
```diff
@@ -0,0 +1,152 @@
...
+    .app-language = Idioma de la aplicación
+    .change-language = Cambiar idioma
+    .theme = Tema
+    .theme-description = Cambia el tema de la aplicación
+    .theme-reset = Restaurar tema
+    .theme-reset-description = Restaurar el tema predeterminado
+    .theme-reset-cta = Limpiar tema
+
+settings-extensions = Extension Settings
+    .placeholder = Marcador de posición
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:98]:
```suggestion
    .placeholder = Espacio Reservado
```

File: shared/locales/es-MX/main.ftl [hunk 31]
```diff
@@ -0,0 +1,152 @@
...
+    .open-extensions-folder = Abrir carpeta de extensiones
+
+settings-privacy = Settings Privacy 
+    .backup-recovery-phrase = Frase de recuperación de copia de seguridad
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:102]:
```suggestion
    .backup-recovery-phrase = Respaldar Frase de Recuperación
```

File: shared/locales/es-MX/main.ftl [hunk 32]
```diff
@@ -0,0 +1,152 @@
...
+    .backup-phrase = Frase de copia de seguridad
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:103]:
```suggestion
    .backup-phrase = Respaldar Frase
```

File: shared/locales/es-MX/main.ftl [hunk 33]
```diff
@@ -0,0 +1,152 @@
...
+    .backup-phrase-description = ¡Haz una copia de esta frase! Junto con tu contraseña, esto representa tu cuenta. Si la pierdes, no podemos ayudarte a recuperarla.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:104]:
```suggestion
    .backup-phrase-description = ¡Haz una copia de esta frase! Junto con tu contraseña, esta representa tu cuenta. Si la pierdes, no podremos ayudarte a recuperarla.
```

File: shared/locales/es-MX/main.ftl [hunk 34]
```diff
@@ -0,0 +1,152 @@
...
+
+settings-audio = Audio Settings 
+    .call-timer = Temporizador de llamada
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:107]:
```suggestion
    .call-timer = Temporizador de Llamada
```

File: shared/locales/es-MX/main.ftl [hunk 35]
```diff
@@ -0,0 +1,152 @@
...
+    .call-timer-description = Cuando está habilitado, se mostrará un temporizador cuando estés en una llamada para mostrar su duración.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:108]:
```suggestion
    .call-timer-description = Cuando está habilitado, se mostrará un temporizador mientras estés en una llamada para mostrar la duración de la misma.
```

File: shared/locales/es-MX/main.ftl [hunk 36]
```diff
@@ -0,0 +1,152 @@
...
+
+settings-files = Files Settings
+    .local-sync = Sincronización Local
+    .local-sync-description = Cuando está habilitado, los archivos se sincronizarán con tu máquina local.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:112]:
```suggestion
    .local-sync-description = Cuando está habilitada, los archivos se sincronizarán con tu máquina local.
```

File: shared/locales/es-MX/main.ftl [hunk 37]
```diff
@@ -0,0 +1,152 @@
...
+    .open-sync-folder = Abrir Carpeta de Sincronización
+    .open-sync-folder-description = Abrir la carpeta donde se sincronizan los archivos.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:114]:
```suggestion
    .open-sync-folder-description = Abrir la carpeta con la que se sincronizan tus archivos.
```

File: shared/locales/es-MX/main.ftl [hunk 38]
```diff
@@ -0,0 +1,152 @@
...
+
+settings-developer = Developer Settings 
+    .developer-mode = Modo de desarrollador
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:117]:
```suggestion
    .developer-mode = Modo de Desarrollador
```

File: shared/locales/es-MX/main.ftl [hunk 39]
```diff
@@ -0,0 +1,152 @@
...
+    .developer-mode-description = Al habilitar el modo de desarrollador, se agrega registro y se muestra información útil de depuración en la interfaz de usuario.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl:118]:
```suggestion
    .developer-mode-description = Al habilitar el modo de desarrollador, se agregan registros y se muestra información útil de depuración en la interfaz de usuario.
```

File: shared/locales/es-MX/main.ftl [hunk 40]
```diff
@@ -0,0 +1,152 @@
...
+media-player = Media Player 
+    .enable-camera = Habilitar Cámara
+    .fullscreen = Pantalla Completa
+    .popout-player = Reproductor de Salida
+    .screenshare = Compartir Pantalla
+    
+remote-controls = Remote Controls 
+    .in-call = En Llamada
+    .mute = Silenciar
+    .unmute = Desilenciar
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .unmute = Desactivar Silencio
```

File: shared/locales/es-MX/main.ftl [hunk 41]
```diff
@@ -0,0 +1,152 @@
...
+    .listen = Escuchar
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .listen = Activar Audio
```

File: shared/locales/es-MX/main.ftl [hunk 42]
```diff
@@ -0,0 +1,152 @@
...
+    .silence = Silencio
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .silence = Desactivar Audio
```

File: shared/locales/es-MX/main.ftl [hunk 43]
```diff
@@ -0,0 +1,152 @@
...
+    .end = Fin
+
+unlock = Unlock
+    .warning1 = Su contraseña se utiliza para cifrar sus datos. Nunca se envía a ningún servidor. Debe usar una contraseña fuerte que no use en ningún otro lugar.
+    .warning2 = Si olvida esta contraseña, no podemos ayudarlo a recuperarla.
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .warning2 = Si olvida esta contraseña, no podremos ayudar a recuperarla.
```

File: shared/locales/es-MX/main.ftl [hunk 44]
```diff
@@ -0,0 +1,152 @@
...
+    .compress-download-cache-description = Para depurar con otros desarrolladores, puedes comprimir tu caché en zip y compartirlo. No hagas esto si esta es una cuenta real que usas.
+    .compress = Comprimir
+    .clear-cache = Limpiar Caché
+    .clear-cache-description = Restablece tu cuenta, básicamente.
+    .clear = Limpiar
+
+media-player = Media Player 
+    .enable-camera = Habilitar Cámara
+    .fullscreen = Pantalla Completa
+    .popout-player = Reproductor de Salida
```
Comment (luisecm) [inline shared/locales/es-MX/main.ftl]:
```suggestion
    .popout-player = Reproductor Emergente
```

**Review by luisecm — COMMENTED** [review]
This is a great job @lgmarchi !!! I have added my suggestions in here. A lot of them were based on how the texts are on the EN translations file, so thats why I am adding a lot of capital-lettered words. And added a few wording changes when I think there are phrases that can be improved.

**Pull request merged**